### PR TITLE
Replace the atomic battery toolmod with atomic battery magazines

### PR DESCRIPTION
**Pull request by Inglonias**
#### Summary
```SUMMARY: Content "Replaces atomic battery toolmods with atomic battery magazines"```

#### Purpose of change
Bring the atomic battery toolmod into the age of battery magazines.

#### Describe the solution
The atomic battery toolmod is no more. In its place are four recipes for the four sizes of atomic battery. The recipes are the same except for the number of plutonium cells required for each one.

Ultralight requires 1 cells
Light requires 2 cells
Medium requires 10 cells
Heavy requires 20 cells.

I'm operating under the assumption that each plutonium cell provides 500 charges of power. Atomic battery cells cannot be recharged, but have a STUPIDLY high power capacity.

#### Describe alternatives you've considered
None, really. Esotericist and I discussed this one. I'll tweak the numbers if they're too absurd, but otherwise I'm pretty happy with this.

#### Additional context
None

### CODE REVIEW

File: data/json/items/ammo.json [hunk 1]
```diff
@@ -36,6 +36,24 @@
     "looks_like": "battery",
     "flags": [ "NO_SALVAGE", "NO_UNLOAD", "RECHARGE" ]
   },
+  {
+    "id": "light_minus_atomic_battery_cell",
+    "type": "MAGAZINE",
+    "category": "spare_parts",
+    "name": "atomic ultra-light battery cell",
+    "description": "This is an atomic battery designed for small size over everything else.  It retains its universal compatibility, though.  It cannot be recharged",
```
Comment (esotericist) [inline data/json/items/ammo.json]: Missing period.

**Comment by esotericist** [comment]
I believe the AFS mod currently has a recipe to manufacture a battery cell using plutonium cells. I can't remember how good/bad the recipe may have been, but it might be worth looking at (and removing) now that we are adding a way of converting plutonium cells to batteries in vanilla.

**Comment by esotericist** [comment]
And since I forgot to say this part: Thanks for getting this done. This looks approximately the right shape (although obviously I don't have final say on the balance implications).

As for this part:
> I'm operating under the assumption that each plutonium cell provides 500 charges of power. Atomic battery cells cannot be recharged, but have a STUPIDLY high power capacity.

Plutonium cells being worth 500 battery charges is actually defined in the c++ code, so this at least is something that is more-or-less fixed in the game at the moment.

**Comment by Night-Pryanik** [comment]
So this is basically the same battery "magazine", just with huge capacity?

**Comment by Inglonias** [comment]
> So this is basically the same battery "magazine", just with huge capacity?

Yup.

**Comment by Night-Pryanik** [comment]
Then it is unclear why is there "atomic" part. The new magazines have nothing "atomic" in them.

**Comment by Inglonias** [comment]
The newly added ones need plutonium cells to craft. Check the recipes.

**Comment by SirPendrak** [comment]
While the idea itself is great, this will make even bigger mess with volume and massratio in game. 
For example, one plutonium cell is 0,05l volume, while ultra light battery has 0,01. I cannot see 10 pcells fitting inside. 
On the other side, light battery, which is 2x more powerfull, is 8 times bigger, standing at 0,08l. 
I suggest making ultra light battery with 1 pultcell, this will make theese 10x stronger, not 100x stronger, and the rest being made proportionally to their volume, so
light ->2pcells->1000power 
medium ->10pcells->5000power
heavy ->20pcells ->10000power

**Comment by Inglonias** [comment]
Definitely not a bad idea in terms of numbers. Makes atomic batteries less ridiculous without affecting their balance.

**Comment by Inglonias** [comment]
Wanted to clarify since there might have been some confusion: When I used "mods" that was short for "toolmods". This is an update to a currently broken item that is present in the base game called the "atomic battery mod" that modifies battery powered tools to run on plutonium cells instead. This PR has nothing to do with game mods.

**Comment by Inglonias** [comment]
I'm waiting for this to merge so I can do some other work on the battery system without having to repeat myself too much.

**Pull request merged**